docs/content/docs/testing/telemetry.md

```diff
@@ -240,24 +242,54 @@ export class ConsoleLogger implements ITelemetryBaseLogger {
 }
 ```
 
-This custom logger is then created and passed into the `getContainer()` and `createContainer()` in a `TinyliciousClient`
-object. The custom logger is now hooked up to the container's telemetry system.
+This custom logger is then created in the `azureConfig` before being passed into `AzureClient`
```

Comment:
the config doesn't create the logger; it's created separately and provided as part of the config

Reply:
good point. I'll rephrase it 😊